.travis.yml

```diff
@@ -1,19 +1,15 @@
 services:
   - docker
-  - redis-server
 
 dist: xenial
 
 before_install:
-  - docker run -d --network host --name zookeeper -e ZOOKEEPER_CLIENT_PORT=2181 confluentinc/cp-zookeeper:4.1.0
-  - docker run -d --network host --name kafka -e KAFKA_ZOOKEEPER_CONNECT=localhost:2181 -e KAFKA_ADVERTISED_LISTENERS=PLAINTEXT://localhost:9092 -e KAFKA_OFFSETS_TOPIC_REPLICATION_FACTOR=1 confluentinc/cp-kafka:4.1.0
-  - docker run -d --net host --name clickhouse-server --ulimit nofile=262144:262144 yandex/clickhouse-server:19.11
-  - make travis-start-redis-cluster
-  - docker build -t getsentry/snuba .
-  - docker ps -a
+  - docker pull getsentry/snuba:latest || true
+  - docker build -t getsentry/snuba . --cache-from getsentry/snuba:latest
```

Comment:
It took me a second to realize what this was doing, but this is pretty clever.

Reply:
We can even cheat further and do
```yaml
- docker pull us.gcr.io/sentryio/snuba:$TRAVIS_COMMIT || docker pull getsentry/snuba:latest || true
- docker build -t getsentry/snuba . --cache-from us.gcr.io/sentryio/snuba:$TRAVIS_COMMIT --cache-from getsentry/snuba:latest
```

If GCB finishes earlier than Travis (quite unlikely) we'd just pull from that image.